tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/BUILD

```diff
@@ -17,6 +17,19 @@ python_tests(
   tags = {'integration'},
 )
 
+python_tests(
+  name='rsc_compile_integration_manual',
```

Comment:
What does manual here mean? Can you add a comment?

Reply:
Ran out idea for names basically, but the idea to use a separate target, so we would have less risk for test timeout if more tests are added to the old target.

`manual` means the test is not auto generated like: https://github.com/pantsbuild/pants/blob/94bc795caf30e2853c399494bd4822832b33a417/tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/test_rsc_compile_integration.py#L97

will clarify.

Reply:
removed the separate target.